scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -34,6 +35,10 @@
 
       .ui-tabs-panel {
         padding: 0 !important;
+
+        div[ng-include] table {
```

Comment:
This selector looks very suspicious, using an angular attribute and two generic tags. There must be a better way.